From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Tue, 17 Mar 2020 16:40:00 +0100
Subject: Atari: Correct sysfs store return

When returning from the sysfs store functions the number of interpreted
bytes should be returned.

Signed-off-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
---
 drivers/hid/hid-atari.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/hid/hid-atari.c b/drivers/hid/hid-atari.c
index 8ca6ff3..6ce8ff8 100644
--- a/drivers/hid/hid-atari.c
+++ b/drivers/hid/hid-atari.c
@@ -548,7 +548,7 @@ static ssize_t xbox_mode_store(struct device *dev,
 	if (v)
 		atari_send_request (ata, ATARI_REQUEST_XBOX_MODE);
 
-	return 0;
+	return count;
 }
 
 static ssize_t calibration_mode_store(struct device *dev,
@@ -570,7 +570,7 @@ static ssize_t calibration_mode_store(struct device *dev,
 	if (v)
 		atari_send_request (ata, ATARI_REQUEST_CALIBRARION_MODE);
 
-	return 0;
+	return count;
 }
 
 static DEVICE_ATTR_RW(leds_firmware_control);
